src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpGrpcExportClient.cs

```diff
@@ -93,7 +93,7 @@ public override ExportClientResponse SendExportRequest(byte[] buffer, int conten
                 grpcStatusDetailsHeader = GrpcProtocolHelpers.GetHeaderValue(trailingHeaders, GrpcStatusDetailsHeader);
             }
 
-            OpenTelemetryProtocolExporterEventSource.Log.ExportFailure(this.Endpoint.ToString(), "Export failed due to unexpected status code.");
+            OpenTelemetryProtocolExporterEventSource.Log.ExportFailure(this.Endpoint.ToString(), "Export failed due to unexpected status code.", (int)status.StatusCode);
```

Comment:
```suggestion
            OpenTelemetryProtocolExporterEventSource.Log.ExportFailure(this.Endpoint.ToString(), $"Export failed due to unexpected status code. {status.ToString()}");
```

Reply:
I would argue against string interpolation here because that's extra work that may get thrown away if no one is subscribed to the event source. But I can change to record the string instead of int.

Reply:
Why not use NonEvent and pass status to it? Status has good detail in it, which covers description.

Reply:
I made a change to include `Status.Detail`. Please re-review.